crud/common/utils.lua

```diff
@@ -381,6 +381,10 @@ function utils.is_uuid(value)
     return ffi.istype(uuid_t, value)
 end
 
+function utils.index_is_allowed(index)
```

Comment:
TREE index helps reducing the number of tuples to scan. Otherwise a full scan is performed. So at least one of condition indexes or primary index should be of type TREE

Reply:
This limitation was added in `select`

Reply:
Done:

```diff
+ -- TREE index helps reducing the number of tuples to scan.
+ -- Otherwise a full scan is performed. So at least one of
+ -- condition indexes or primary index should be of type TREE
function utils.index_is_allowed(index)
    return index.type == 'TREE'
end
```